doc/installation.md

```diff
@@ -0,0 +1,98 @@
+## Introduction
+
+The workflow of the installation is customized via the [control
+file](https://github.com/yast/yast-installation/blob/master/doc/control-file.md), 
+and below we have the principal steps concerning network configuration:
+
+  - Linuxrc
+  - inst_install_inf (What is still needed)
+  - inst_dhcp
+  - manual_configuration (openSUSE only in Addons)
```

Comment:
In SLE you can launch the `lan` client manually in `disks_activation`, `registration` and `addons' like you can see [here](https://gist.github.com/teclator/35a6aee2cd387605b6ef#final-implementation), so basically is the same without `registration`. I will rewrite it.